### PR TITLE
ign -> gz Upstream Macro Migration : sdformat

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
LGTM with 🟢 CI

File: src/ign.hh [hunk 1]
```diff
@@ -34,7 +34,8 @@ extern "C" SDFORMAT_VISIBLE int cmdCheck(const char *_path);
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" SDFORMAT_VISIBLE char *ignitionVersion();
+extern "C" SDFORMAT_VISIBLE char SDF_DEPRECATED(13) *ignitionVersion();
```
Comment (chapulina) [inline src/ign.hh:37]: This header isn't installed, so we don't need to tick-tock. Just changing to `gz` is enough.

File: Migration.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ but with improved human-readability..
 - **sdf/Types.hh**: The `Inertia` class has been deprecated. Please use the
 ￼   `Inertial` class in the `gz-math` library.
 
+- `ignitionVersion()` is deprecated and will be removed in future versions.
+  Use `gzVersion()` instead.
```
Comment (chapulina) [inline Migration.md:29]: Once we remove the tick, we can also remove this note

**Review by chapulina — APPROVED** [review]
LGTM, we should just revert the tick-tock of a private function

File: src/ign.hh (hunk 1 above)
Comment (methylDragon) [reply src/ign.hh]: https://github.com/gazebosim/sdformat/pull/1040/commits/ac77e5ea225ce755a66357d1ed2bff935bd789f5 !

File: Migration.md (hunk 2 above)
Comment (methylDragon) [reply Migration.md]: https://github.com/gazebosim/sdformat/pull/1040/commits/ac77e5ea225ce755a66357d1ed2bff935bd789f5 !

**Review by chapulina — APPROVED** [review]

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**